modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/endlos.cmd

```diff
@@ -0,0 +1,4 @@
+REM Keep restarting docker to repro a certificate creation issue on windows containers
+REM Build the container with dockerbuild.cmd
+docker run opcpublisher:latest OpcPublisher >>endlesstest.log
```

Comment:
Do we need this script here given how test-case specific it is ?

Reply:
normally I would not check it in, but since this is a very special issue and difficult to repro better keep itz. And its not in main

Reply:
OK, agreed.